apps/ftu/test/unit/ui_manager_test.js

```diff
@@ -516,4 +516,64 @@ suite('UI Manager > ', function() {
     });
   });
 
+  suite('Metrics Section', function() {
+    var MockSettings;
+
+    setup(function() {
+      MockNavigatorSettings.mSetup();
+      MockSettings = MockNavigatorSettings.mSettings;
+      UIManager.init();
+      Navigation.currentStep = 7;
```

Comment:
This should be currentStepIndex, currentStep is expected to be an object reference with the hash etc.